### ISSUE TITLE
Add spectrum analyser / spectrometer

### ISSUE THREAD

**Issue by MagicLike** [geoffreybennett/alsa-scarlett-gui#140]
# Add spectrum analyser / spectrometer

## Confirmation
- [x] I confirm that I have read the [FAQ](https://github.com/geoffreybennett/alsa-scarlett-gui/blob/master/FAQ.md).

## Issue Category
Please select the category that best describes your issue:
- [ ] Help Request
- [ ] Bug Report
- [x] Feature Request

## Feature Description
In addition to the `Levels` view a kind of "spectrum analyser" / spectrometer showing the different frequencies being hit would imo be a great (and cool) addition

**Comment by geoffreybennett** [comment]
Yeah, a spectrum analyser isn't within the scope of this application, until you convince Focusrite to run the spectrum analyser on the Scarlett's DSP & alsa-scarlett-gui will display the results. But at least at the moment, the correct solution is software such as: https://lsp-plug.in/?page=plugins or https://x42-plugins.com/x42/x42-meters
